src/index.js

```diff
@@ -0,0 +1,49 @@
+import './style.css';
+import Store from './modules/locolstorage.js';
+import ToDoInfo from './modules/add.js';
+import { check } from './modules/status.js';
+
+// added input to active event listener
+const input = document.getElementById('input');
+document.addEventListener('DOMContentLoaded', ToDoInfo);
+input.addEventListener('keypress', (event) => {
+  if (event.key === 'Enter') {
+    if (input.value === '') return;
+    event.preventDefault();
+    const addedwork = input.value;
+    const retrieveData = localStorage.getItem('list');
+    // created the index for Local storage
+    let ind = 1;
```

Comment:
Try to follow the variable name in a way that reveals the intention behind it. You can see more details regarding the same on [here](https://devinduct.com/blogpost/22/javascript-clean-code-best-practices)